src/utils/data-constants.js

```diff
@@ -22,10 +22,41 @@ export const HOUSING_TYPES = [
   '3-BR',
 ];
 
+export const NEIGHBORHOODS = [
+  undefined,
+  '122nd-Division',
+  'Belmont-Hawthorne-Division',
+  'Centennial-Glenfair-Wilkes',
+  'Central City',
+  'Forest Park-Northwest Hills',
+  'Gateway',
+  'Hayden Island',
+  'Hillsdale-Multnomah-Barbur',
+  'Hollywood',
+  'Interstate Corridor',
+  'Lents-Foster',
+  'MLK-Alberta',
+  'Montavilla',
+  'Northwest',
+  'Parkrose-Argay',
+  'Pleasant Valley',
+  'Raleigh Hills',
+  'Roseway-Cully',
+  'Sellwood-Moreland-Brooklyn',
+  'South Portland-Maraquam Hill',
+  'St. Johns',
+  'Tryon Creek-South Terwilliger',
+  'West Portland',
+  'Woodstock',
+  'Portland',
+];
```

Comment:
We should get the name from the data provided to the map instead of re-associating it by means of this constant.